HybridCar.cpp

```diff
@@ -1,15 +1,16 @@
 #include "HybridCar.hpp"
 #include <iostream>
 
-HybridCar::HybridCar(PetrolEngine* petrolEng, ElectricEngine* electricEng)
-    : PetrolCar(petrolEng)
-    , ElectricCar(electricEng)
+HybridCar::HybridCar(PetrolEngine* petrolEng, ElectricEngine* electricEng, int vmax)
+    : PetrolCar(petrolEng, vmax)
+    , ElectricCar(electricEng, vmax)
 {
+    Car::Vmax = vmax;
```

Comment:
kazdy konstruktor ustawia Car::Vmax
lepiej dodac konstruktor do Car jaki to zrobi